### PR TITLE
Max 12

### PR DESCRIPTION
**Pull request by MassimoVilla**
Here comes a new manuscript. MS BDLaethg31 is a small codex containing two apocryphal texts and one liturgical text. Two of them were previosuly unrecorded in BM.

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethg31.xml [hunk 1]
```diff
@@ -10,10 +10,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Golgotā, ʾArdǝʾǝt (?), Ṣalota fǝtḥata māy</title>
+                <title xml:lang="gez" xml:id="title1">ጸሎት፡ ዘእግዝእትነ፡ ማርያም፡ በደብረ፡ ጎልጎታ፡,</title>
+                <title xml:lang="gez" corresp="#title1" type="normalized">Ṣalot za-ʾǝgzǝʾǝtǝna Māryām ba-Dabra Golgotā , ʾArdǝʾǝt (?), Ṣalota fǝtḥata māy</title>
+                <title xml:lang="en" corresp="#title1"> Prayer of Our Lady Mary on Mount Golgotha, </title>
```
Comment (gnizla) [inline OxfordBodleian/Juel-Jensen/BDLaethg31.xml]: Not doing proofreading, as we discussed last meeting, but is the comma intentional?

**Comment by gnizla** [comment]
Thank you "King" Max (as Solomon calls you!)

**Review by ElianaDalSasso — APPROVED** [review]
Thank you Massimo! Just a minor comment on the bindingDesc of g. 31. However, I need to look at it in detail.

File: OxfordBodleian/Juel-Jensen/BDLaethg31.xml [hunk 2]
```diff
@@ -123,55 +189,112 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                        A stanza has been added in a secondary hand in the lower margin of <locus target="#25v"/> and in the upper margin of <locus target="#26r"/>.
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
+                        
                         <bindingDesc>
                             <binding xml:id="binding">
-                                <decoNote xml:id="b1">Dark red leather cover. Edges gilded with floral pattern. Red guard leaves (Seidenpapier) textured and shining.</decoNote>
+                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> boards covered with dark red <material key="leather"/>.
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethg31.xml:272]: I am unable to judge from the pictures, but the pasteboard option seems now preferable to me too.

**Review by thea-m — CHANGES_REQUESTED** [review]
Thanks a lot! A few minor remarks, mostly encoding-related.

File: OxfordBodleian/Juel-Jensen/BDLaethg31.xml [hunk 3]
```diff
@@ -37,10 +41,68 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
-                        
+                     <msItem xml:id="ms_i1">
+                            <locus from="1r" to="31v"/>
+                         <title type="complete" ref="LIT2246salotz"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez" type="supplication">
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethg31.xml:48]: my mistake: I reduplicated the `<incipit xml:lang="gez" type="supplication">` element without changing/removing "supplication"

File: OxfordBodleian/Juel-Jensen/BDLaethg31.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethg31.xml]: P.S. I am seeing that "pasteboard" is not allowed as a value in `<material key="pasteboard"/>`. Should it be added to the allowed values in this position?

File: OxfordBodleian/Juel-Jensen/BDLaethg31.xml [hunk 4]
```diff
@@ -123,55 +189,112 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                        <!-- <decoDesc>
-                         
                         </decoDesc>-->
                         
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="Unclear"/>                                      
+                                    <locus target="#31v"/>
+                                    A poorly understandable note has been written in an untrained hand. The text reads as follows: <q xml:lang="gez">ወልደ፡ ጊዮርጊስ፡ ምንሱቅ<supplied reason="omitted">፡</supplied>
```
[the remark this answers is not in the document]
Comment (MassimoVilla) [reply OxfordBodleian/Juel-Jensen/BDLaethg31.xml:236]: Good! Please check if now the description of a1 is formulated properly.

File: OxfordBodleian/Juel-Jensen/BDLaethg31.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ElianaDalSasso) [reply OxfordBodleian/Juel-Jensen/BDLaethg31.xml]: Yeah, you're right it needs to be added. I used to create keywords but I think that here the process is different and might involve changes to the ODD. Am I wrong? Does anyoneone know how to do it?

**Comment by thea-m** [comment]
Thank you! It's fine by me, let's keep it open until the new value exists, so that we don't get in the habit of using non-existent values :) Maybe in the meanwhile Solomon might have some input on the note.

**Comment by thea-m** [comment]
I was about to open the issue and then noticed that we do have the value "cardboard" - is that the same as pasteboard (or is it used in the same way in the existing records)?

**Comment by ElianaDalSasso** [comment]
Sorry for the delay! 
- @thea-m I explored a little the Ligatus terminology. It seems that pasteboard is now considered an improper term to refer to boards made from pasted laminations of paper (http://w3id.org/lob/concept/1492). Then there is Card, a term " without a specific definition" used "to describe a relatively stiff thick paper or a thin paper board" http://w3id.org/lob/concept/4614. They do not define cardboard....but the Oxford English Dictionary defines it as: "Thick, stiff paper or pasteboard, used for making boxes, etc. Also (now rare): a piece of such material". From what I know, the boards made of paper can be formed either by sheets of paper pasted together (pasteboard) or by a thick pulp of paper fibres (pulp board)...and here arises the confusion.
In the end, the terms cardboard, pasteboard, pulp board, pasted-laminated boards seem very (too much) specific to me. I am wondering if the term 'paper' could be added as a value instead. This would permit to say: 
`<decoNote xml:id="b1">Two boards made from pasted laminations of <material key="paper">paper</material> ` or
`<decoNote xml:id="b1">Two boards made of <material key="paper">paper</material> pulp` perhaps also
`<decoNote xml:id="b1">Two boards made of stiff thick <material key="paper">paper</material>` or 
`<decoNote xml:id="b1">Two  <material key="paper">paper</material> boards` would work.
- However, I was thinking that I am just supposing that the boards are not made of wood based on the style of the binding but having not seen them in person, I could also be wrong. 
- @MassimoVilla I can do it! I have to go through all bindingDesc anyway.

**Comment by thea-m** [comment]
Thank you! I opened an issue about the general use of "cardboard". Regardless of that, the term "paper" already exists, so that might be a good solution here?

**Comment by ElianaDalSasso** [comment]
No bothering at all! The discussion is still opened in the issue #2009 open by @thea-m. We could use tentatively use the value "paper" until the issue is solved!

**Comment by thea-m** [comment]
Thanks @MassimoVilla and everybody!

**Pull request merged**